app/src/main/java/com/example/parkingsystem/utilities/Picker.java

```diff
@@ -14,7 +14,7 @@
 import static java.util.Calendar.MINUTE;
 
 public class Picker {
-    public DatePickerDialog datePickerDialog;
+    private DatePickerDialog datePickerDialog;
     Date date;
```

Comment:
"date" field still doesn't specify access.

Reply:
I believe I resolved the conversation before pushing the changes, but I changed the Date to private Date.

Reply:
I believe I resolved this before pushing the changes but I did specified the access as private, not sure why it does not appear.